index.html

```diff
@@ -11,14 +11,47 @@
         <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;700&display=swap" rel="stylesheet">
     </head>
     <body>
-  <header> 
-      <h1>Two Truths, One Lie</h1>
-  </header>
-  <main>
-    <!-- how will you mark up your media objects -->
-  </main>
-  <footer>
-    <h3>By YOUR NAMES HERE</h3>
-  </footer>
+      <header>
+        <h1>Two Truths, One Lie</h1>
+        <p>Can you guess which statement is false?</p>
+      </header>
+      <main class="main">
+        <!-- how will you mark up your media objects -->
+        <section class="azan">
+          <div class="name">
+            <img src="img/download (1).jpg" />
```

Comment:
Your app is failing Accessibility in Lighthouse because you do not have alt attributes in your images.